src/server/scripts/Commands/cs_debug.cpp

```diff
@@ -579,13 +514,8 @@ class debug_commandscript : public CommandScript
         return true;
     }
 
-    static bool HandleDebugGetItemStateCommand(ChatHandler* handler, char const* args)
+    static bool HandleDebugGetItemStateCommand(ChatHandler* handler, std::string itemState)
```

Comment:
keep in mind that this only gets the input up to the first whitespace - is this intended?

Reply:
I'd say so. The expected possible strings are all without whitespace.